src/App.js

```diff
@@ -1,5 +1,105 @@
+import { Console } from "@woowacourse/mission-utils";
+import { Random } from "@woowacourse/mission-utils";
 class App {
-  async run() {}
+  parseCarNames(carNames) {
+    return carNames.split(",").map((carName) => {
+      return { name: carName.trim(), result: "" };
+    });
+  }
+
+  validateCarNames(carNames) {
+    carNames.split(",").forEach((carName) => {
+      if (!carName.trim()) {
+        throw new Error("[ERROR]: 자동차 이름을 입력해주세요.");
+      }
+      if (carName.trim().length > 5) {
+        throw new Error("[ERROR]: 자동차 이름은 5자 이하만 가능합니다.");
+      }
+    });
+  }
+
+  validateAttempts(attempts) {
+    if (!attempts.trim()) {
+      throw new Error("[ERROR]: 시도 횟수를 입력해주세요.");
+    }
+
+    if (isNaN(attempts)) {
+      throw new Error("[ERROR]: 시도 횟수는 숫자만 입력 가능합니다.");
+    }
+
+    if (attempts < 1) {
+      throw new Error("[ERROR]: 시도 횟수는 1 이상만 가능합니다.");
+    }
+  }
+
+  async getCarNameAndAttempts() {
+    try {
+      const carName = await Console.readLineAsync(
+        "경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)\n"
+      );
+      this.validateCarNames(carName);
+      const attempts = await Console.readLineAsync(
+        "시도할 횟수는 몇 회인가요?\n"
```

Comment:
Airbnb 스타일 가이드에 따르면 문자열을 감쌀 때 작은따옴표를 사용하라고 권장하고 있어요!

한번 확인해보시면 좋을 것 같아요😄

[Airbnb JavaScript Style Guide : strings--quotes](https://github.com/airbnb/javascript?tab=readme-ov-file#strings--quotes)
```suggestion
        '시도할 횟수는 몇 회인가요?\n'
```